docker-compose.yml

```diff
@@ -73,6 +74,7 @@ services:
       - BB_CLIENT_KEY_FILE=../shared_files/bb-dev-test-key.pem 
       - BB_SERVER_LOCATION=https://fhir.backend.bluebutton.hhsdevcloud.us
       - BB_TIMEOUT_MS=500
+      - BCDA_FHIR_MAX_RECORDS=10000
```

Comment:
I didn't see this env var being used by `bcdaworker`.  Is this needed?

Reply:
I wasn't sure.  I would rather add it now than be super confused about what is happening later when it isn't there.

Reply:
I think the answer is... if you use it in the `bcdaworker` code, then it should be here.  Otherwise, it shouldn't.  This is important because this is where we usually grab the env vars required for S3/AWS.